src/Server.cpp

```diff
@@ -135,18 +135,15 @@ void Server::handleConnection(Connection conn, size_t &i)
 	HTTPRequest request(httpRequestString.c_str());
```

Comment:
It would be good to extract this code (form probably 138 till 196) from handleConnection and create a method for the routing and response handling that will be called a this place. What do you think?

Reply:
Yes that was one of the goals of PR 147 and has been done